server/src/models/index.js

```diff
@@ -0,0 +1,13 @@
+// const dbConfig = require('../.env.example');
+
+const mongoose = require('mongoose');
+const mongoosePaginate = require('mongoose-paginate-v2');
+
+mongoose.Promise = global.Promise;
```

Comment:
quick googling shows that this is really legacy code. Did you simply copy paste this whole section? https://stackoverflow.com/questions/51862570/mongoose-why-we-make-mongoose-promise-global-promise-when-setting-a-mongoo

Reply:
Removed